vcd_cli/org.py

```diff
@@ -139,3 +143,37 @@ def create(ctx, name, full_name):
         stdout('Org \'%s\' is successfully created.' % result.get('name'), ctx)
     except Exception as e:
         stderr(e, ctx)
+
+
+@org.command(short_help='delete an organization')
+@click.pass_context
+@click.argument('name',
+                metavar='<name>',
+                required=True)
+@click.option('-r',
+              '--recursive',
+              is_flag=True,
+              help='removes the Org and any objects it contains that are in a '
+                   'state that normally allows removal')
+@click.option('-f',
+              '--force',
+              is_flag=True,
+              help='pass this option along with --recursive to remove an Org'
+                   ' and any objects it contains, regardless of their state')
+@click.option('-y',
+              '--yes',
+              is_flag=True,
+              callback=abort_if_false,
+              expose_value=False,
+              prompt='Are you sure you want to delete the org?')
```

Comment:
"Are you sure you want to delete the Org, it will force delete all of its entities regardless of the their state?" - Descriptive warning?

Reply:
I wanted to warn about force delete only when --force option is used.

C:\sources\github\service_ecosystem\vcd-cli (master) 
(vcd-cli) λ vcd org delete Test1
Are you sure you want to delete the Org? [y/N]: y
task: 93cf9e36-d9d7-44f1-9dbd-76e528e6dd01, ORG_DELETE_ORG Organization Test1(195a630f-19e4-4341-a721-dd00204dba2e), result: success

C:\sources\github\service_ecosystem\vcd-cli (master) 
(vcd-cli) λ vcd org delete Test1 --force
Are you sure you want to delete the org? [y/N]: y
Do you want to force delete 'Test1' [y/N]: y
task: ac747fd1-d880-40ce-8187-4632976b8f1e, ORG_DELETE_ORG Organization Test1(cbf75852-5d11-4263-abe7-82da9047dbb9), result: success

C:\sources\github\service_ecosystem\vcd-cli (master) 
(vcd-cli) λ vcd org delete org1 --force --recursive
Are you sure you want to delete the org? [y/N]: y
Do you want to force delete 'org1' and all its objects recursively? [y/N]: y
task: 296331ed-6de9-466a-899f-0bd6295673ff, ORG_DELETE_ORG Organization org1(7b4c5110-a41a-4ae5-92ad-bc7814d40055), result: success